spec/displaynames.html

```diff
@@ -0,0 +1,263 @@
+<emu-biblio href="./biblio.json"></emu-biblio>
+<emu-clause id="intl-displaynames-objects">
+  <h1>DisplayNames Objects</h1>
+
+  <emu-clause id="sec-intl-displaynames-abstracts">
+    <h1>Abstract Operations for DisplayNames Objects</h1>
+
+    <emu-clause id="sec-canonicalcodefordisplaynames" aoid="CanonicalCodeForDisplayNames">
+      <h1>CanonicalCodeForDisplayNames ( _type_, _code_ )</h1>
+      <p>
+        The CanonicalCodeForDisplayNames abstract operation is called with arguments _type_, and _code_. It verifies that the _code_ argument represents a well-formed code according to the _type_ argument and returns the case-regularized form of the _code_. The algorithm refers to <a href="https://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>. The following steps are taken:
+      </p>
+      <emu-alg>
+        1. If _type_ is `"language"`, then
+          1. If _code_ does not matches the `unicode_language_id` production, throw a *RangeError* exceptipon.
```

Comment:
```suggestion
        1. If _type_ is *"language"*, then
          1. If _code_ does not match the `unicode_language_id` production, throw a *RangeError* exception.
```

Following patterns with 262, wrap string representations with star and double quotes, e.g. `*"string"*` instead the backtick quotes.